spec/system/task_spec.rb

```diff
@@ -1,24 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`を設定することで`project_id`の指定は必要なくなりますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
607f8bc
associationを追加しました